fearless/Modules/Staking/StakingMain/ViewModel/StakingViewState.swift

```diff
@@ -3,7 +3,10 @@ import SoraFoundation
 
 enum StakingViewState {
     case undefined
-    case nominator(viewModel: LocalizableResource<NominationViewModelProtocol>)
+    case nominator(
+        viewModel: LocalizableResource<NominationViewModelProtocol>,
+        alerts: [StakingAlert]
+    )
     case validator(viewModel: LocalizableResource<ValidationViewModelProtocol>)
```

Comment:
I think we need we need alerts for other states too (except undefined). For now let's apply only ```election period``` to them.